src/main/g8/.scalafmt.conf

```
@@ -1,2 +1,8 @@
 version = 3.4.3
```

... while we're here :)

```suggestion
version = 3.7.1
```

---

Oh... I guess steward doesn't bump this?